Add docs about PACKIT_TAG_NAME (#918)
packit/packit.dev: docs/configuration/upstream/tests.md

* `PACKIT_BUILD_LOG_URL`
* `PACKIT_SRPM_URL`
* `PACKIT_COMMIT_SHA`

> * `PACKIT_TAG_NAME`, tag name from release event, not set if Testing Farm wasn't triggered by ReleaseEvent

Reviewer: ```suggestion
* `PACKIT_TAG_NAME`, tag name from release event, not set if the test job doesn't have **release** trigger
```
